RadarCovid/Config/Config.swift

```diff
@@ -12,71 +12,39 @@
 import Foundation
 import DP3TSDK
 
-enum Endpoits {
-    case pre
-    case pro
-    var config: String {
-        switch self {
-        case .pre: return "https://radarcovidpre.covid19.gob.es/configuration"
-        case .pro: return "https://radarcovid.covid19.gob.es/configuration"
-        }
-    }
-    var kpi: String {
-        switch self {
-        case .pre: return "https://radarcovidpre.covid19.gob.es/kpi"
-        case .pro: return "https://radarcovid.covid19.gob.es/kpi"
-        }
-    }
-    var dpppt: String {
-        switch self {
-        case .pre: return "https://radarcovidpre.covid19.gob.es/dp3t"
-        case .pro: return "https://radarcovid.covid19.gob.es/dp3t"
-        }
-    }
-    var verification: String {
-        switch self {
-        case .pre: return "https://radarcovidpre.covid19.gob.es/verification"
-        case .pro: return "https://radarcovid.covid19.gob.es/verification"
-        }
-    }
-}
-
-
-
 struct Config {
-    
+    static let userDefaultsSuiteName = "es.gob.radarcovid.userdefaults.suite"
```

Comment:
Shouldn't this be `group.es.gob.radarcovid` so `UserDefaults` can be shared with app extensions using the App Group?

Reply:
My bad!

However, since the group created between the app and the widget is also fictitious since it has to be created in the development account, this variable can also be changed without prejudice to anything else.

Anyway change is on the way

Reply:
Another related detail that is not very important currently as the App Group is fictitious: the App Group entitlement is also missing in the app, it is only assigned to the app extension.

In any case, great contribution. Thanks!

Reply:
Fixed on last commit. Thank you for your support!